src/common/slack.ts

```diff
@@ -244,11 +244,13 @@ export const getSlackIntegration = async ({
   return slackIntegration;
 };
 
-export const getSlackIntegrationOrFail: typeof getSlackIntegration = async ({
+export const getSlackIntegrationOrFail = async ({
   id,
   userId,
   con,
-}) => {
+}: Parameters<
+  typeof getSlackIntegration
+>[0]): Promise<UserIntegrationSlack> => {
```

Comment:
Kinda ugly, but guess it's what it is

Reply:
Agreed, added `PropsParameters` util so we can extract more easily also looks better 6f0ad7d65c4763b795315d879f6d1bd424390cf4